Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -51,32 +51,37 @@ def bump_formula_pr
       safe_system "brew", "update"
     end
 
-    Utils::Inreplace.inreplace(formula.path) do |s|
-      if new_url_hash
-        old_url = formula_spec.url
-        if ARGV.dry_run?
-          ohai "replace '#{old_url}' with '#{new_url}'"
-          ohai "replace '#{old_hash}' with '#{new_hash}'"
-        else
-          s.gsub!(old_url, new_url)
-          s.gsub!(old_hash, new_hash)
-        end
-      else
-        resource_specs = formula_spec.specs
-        old_tag = resource_specs[:tag]
-        old_revision = resource_specs[:revision]
-        if ARGV.dry_run?
-          ohai "replace '#{old_tag}' with '#{new_tag}'"
-          ohai "replace '#{old_revision}' with '#{new_revision}'"
-        else
-          s.gsub!(/['"]#{old_tag}['"]/, "\"#{new_tag}\"")
-          s.gsub!(old_revision, new_revision)
-        end
-      end
+    s = (formula.path.open("r") { |f| Formulary.set_encoding(f).read }).extend(StringInreplaceExtension)
+    old_formula_version = Formulary.from_contents(formula.name, formula.path, s, requested_spec).version
```

Comment:
Works for me. Though since `old_formula_version` is exactly as long as `old_formula.version`, using `old_formula.version` directly seems more desirable than creating a separate variable.